### DISCUSSION TITLE
Theme

### DISCUSSION THREAD

**Discussion by vi-cat** [groepl/Obsidian-Templates#15] [Q&A] [answered]
In the screenshots that you have shared in the repo about your Obsidian setup:

1. What theme do you use?
2. How did you get the cute arrows as icons in links?

Thanks 🙏

**Comment by groepl** [comment, marked as answer]
Sorry for the later answer. 🥴

![IMG_1351](https://github.com/user-attachments/assets/dd1387c2-427b-4159-9afc-f59aaa9e52ae)

I’m using the Default themes from Obsidian, but with an individual accent color.

**Reply by groepl** [reply]
The icons are also part of the Default themes.
